x/commitment/keeper/keeper.go

```diff
@@ -6,18 +6,37 @@ import (
 	"github.com/cosmos/cosmos-sdk/codec"
 	storetypes "github.com/cosmos/cosmos-sdk/store/types"
 	sdk "github.com/cosmos/cosmos-sdk/types"
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
 	paramtypes "github.com/cosmos/cosmos-sdk/x/params/types"
 	"github.com/tendermint/tendermint/libs/log"
 
 	"github.com/elys-network/elys/x/commitment/types"
 )
 
+// Interface declearation
+type CommitmentKeeperI interface {
```

Comment:
Could you put this interface on  types/expected_keepers.go

Reply:
This is interface declaration not reference.